### PR TITLE
Initial question about risk factors

### PR DESCRIPTION
**Pull request by sjain1701**
That's the general area I want to look into for the project.

### CODE REVIEW

**Comment by chendaniely** [comment]
Thanks for submitting the PR! :tada: 

A few things:

1. Can you rename the file so it's `README.md` this way it'll show up and render automatically when the PR gets merged.
    - When you rename it, `git` will treat it as a deleted file + new file. When you `add` both of them it will recognize it as a file rename (or move). Then `commit` and `push` just like you did for the PR. It will automatically update this PR, no need to resubmit a new one.

2. Looking for mentioned risk factors is a good project (right @ambrown77 ?). A few things to think about (and write in the `README.md`:
    - How do you plan on identifying "risk factors"? I agree working with the abstract is a good way to reduce the text you will do for analysis, but once you have the text how will you know what is a risk factor? Do you plan to read through and manually curate?
    - Something that might help is looking at `n-grams`, but I'm not sure where you'd go from there

**Comment by chendaniely** [comment]
Some more thoughts/ideas:

since "risk factors" are going to be nouns, I *think* we can part-of-speech tag the words in the abstract and look for "nouns". I'm not sure how that will work for risk factors that are multiple words, e.g., "shortness of breath" but that's where n-grams might be able to help.

An even more basic version of this would then look at the tf-idf of the words in the abstract, and see what "important" words pop up and then filter them down by nouns or something...

maybe? this will work?

File: analysis/db/somya/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+I was thinking about looking into risk factors for COVID-19. Maybe which ones are most talked about or looked into?
+Something simple to start with might be looking at just the abstracts of a few files and seeing what risk factors they mention and then keeping track of the count.
+I would like to use the risk factors that they identified on [Kaggle](https://www.kaggle.com/allen-institute-for-ai/CORD-19-research-challenge/tasks?taskId=558).
+Specifically, I would like to start with smoking, co-infections, and pregnancy.
```
Comment (ambrown77) [inline analysis/db/somya/README.md:4]: Good thinking with risk factors! I believe diabetes, heart disease, etc are ones to also think about. Can you identify all papers in the data set with risk factors mentioned in them, pull what they are, and then determine associations from that? Also, are there any additional datasets published on risk factors?

File: analysis/db/somya/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+I was thinking about looking into risk factors for COVID-19. Maybe which ones are most talked about or looked into?
+Something simple to start with might be looking at just the abstracts of a few files and seeing what risk factors they mention and then keeping track of the count.
+I would like to use the risk factors that they identified on [Kaggle](https://www.kaggle.com/allen-institute-for-ai/CORD-19-research-challenge/tasks?taskId=558).
```
Comment (ambrown77) [inline analysis/db/somya/README.md:3]: Good thinking with risk factors! I believe diabetes, heart disease, etc are ones to also think about. Can you identify all papers in the data set with risk factors mentioned in them, pull what they are, and then determine associations from that? Also, are there any additional datasets published on risk factors?

**Comment by chendaniely** [comment]
@sjain1701 can you add a link to this issue~ (https://github.com/databridgevt/covid19/pull/10) in your file? that way we won't loose/forget about the conversation here. I'll merge it in right after. :+1:

**Comment by chendaniely** [comment]
nvm I did it for you in bedd212a20817407ba049ba34b6913f760861756

**Comment by chendaniely** [comment]
please run the following to clean up the branches on your computer:

```
git fetch --all --prune
git checkout master
git pull origin master
git branch -d somya_ques
```

**Pull request merged**